doc/manual/noteworthy-differences.rst

```diff
@@ -251,3 +251,75 @@ Noteworthy differences from Python
   returns a new random number every time it is invoked without argument. On the
   other hand, the function ``g(x=[1,2]) = push!(x,3)`` returns ``[1,2,3]`` every
   time it is called as ``g()``.
+
+
+Noteworthy differences from C/C++
+----------------------------------
+
+- Julia arrays are indexed with square brackets, and can have more than one
+  dimension ``A[i,j]``.
+- In Julia, indexing of arrays, strings, etc. is 1-based not 0-based.
+- Julia arrays are assigned by reference. After ``A=B``, changing elements of
```

Comment:
I guess saying `arrays` is probably too limited for C/C++ section, because everything is assigned by reference. I think it was appropriate in the MATLAB section because everything in MATLAB is basically an array.